File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robot/Pivot.java
package org.firstinspires.ftc.teamcode.robot;

import com.qualcomm.robotcore.hardware.DcMotor;
import com.qualcomm.robotcore.hardware.HardwareMap;

public class Pivot {
    private final DcMotor pivot;

    public Pivot(HardwareMap hardwareMap) {
        pivot = hardwareMap.dcMotor.get("pivot");
        pivot.setMode(DcMotor.RunMode.RUN_WITHOUT_ENCODER);
        pivot.setZeroPowerBehavior(DcMotor.ZeroPowerBehavior.BRAKE);
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot/Pivot.kt
package org.firstinspires.ftc.teamcode.Robot

import com.qualcomm.robotcore.hardware.DcMotor
import com.qualcomm.robotcore.hardware.DcMotorSimple
import com.qualcomm.robotcore.hardware.HardwareMap

class Pivot(hardwareMap: HardwareMap) {

    //    var targetPosition: Int
    private val pivot = hardwareMap.dcMotor.get("pivot").apply {
        mode = DcMotor.RunMode.RUN_WITHOUT_ENCODER
        zeroPowerBehavior = DcMotor.ZeroPowerBehavior.BRAKE
    }

}